Algorithms/UnionFind.java

```diff
@@ -0,0 +1,81 @@
+
+
+/**
+ * @author johnny16
+ *
+ */
```

Comment:
Hey, could you please remove this comment

Reply:
Done

Reply:
Have you pushed the commit with the recent changes?